lib/matplotlib/lines.py

```diff
@@ -592,6 +599,59 @@ def get_markevery(self):
         """return the markevery setting"""
         return self._markevery
 
+    def set_downsample(self, downsample):
```

Comment:
I mean add down sample as a property (ex `downsample = property(get_downsample, set_downsample)`).  We are in the mid-term future moving away from all of the getter/setter methods.